Updated readme (#781)
zabbix/zabbix-docker: README.md

* [zabbix-java-gateway](https://hub.docker.com/r/zabbix/zabbix-java-gateway/) - Zabbix Java Gateway
* [zabbix-snmptraps](https://hub.docker.com/r/zabbix/zabbix-snmptraps/) - Additional container image for Zabbix server and Zabbix proxy to support SNMP traps

* [zabbix-appliance](https://hub.docker.com/r/zabbix/zabbix-appliance/) - **deprecated, please do not use this any longer** Zabbix appliance with built-in MySQL server, Zabbix server, Zabbix Java Gateway and Zabbix frontend based on Nginx web-server

## Deploy via docker-compose

### Usage

Instead of using zabbix-appliance that was packing every service into one container you should now use the provided docker-compose files.

> 1. pull the repository

Reviewer: ```suggestion
1. Pull the repository.
```

> 2. rename the docker-compose-os-db-buildtype.yaml file you want to docker-compose.yaml

Reviewer: ```suggestion
2. Choose the docker-compose file you want to use and remove / comment extra services (for example proxy) you don't want.
```

Reviewer: Renaming is unnecessary and it's better to keep the original file in my opinion.
So that you can keep up with latest git updates easily.

> 3. review the .env and .User / Password files

Reviewer: ```suggestion
3. Review the `.env` and ._DATABASE__USER/PASSWORD files.
```

> 3. run `docker-compose up -d` from inside the directory

Reviewer: ```suggestion
3. Run `docker-compose -f YOUR_SUITABLE_DOCKER_COMPOSE up -d` from inside the directory.
```

> 4. Have your full stack zabbix instance available at localhost:80

Reviewer: ```suggestion
4. Have your full stack zabbix instance available at localhost:80.
```
